docs/_data/supporters.js

```diff
@@ -139,7 +165,15 @@ module.exports = async () => {
     supporters.backers.map(async backer => {
       const filePath = resolve(supporterImagePath, backer.id + '.png');
       const {body} = await needle('get', encodeURI(backer.avatar));
-      await writeFile(filePath, body);
+
+      let imageBuffer = body;
+
+      // Some times OpenCollective responds with a non-image
+      if ((await fileType.fromBuffer(body)).mime !== 'image/png') {
```

Comment:
so yeah, with my changes, _`canvas`_ is doing the file checking.  currently, if it fails, it just discards the image, but you could replace that with this that just uses a blank one... or not.